src/components/MovieDetails/MovieDetails.tsx

```diff
@@ -0,0 +1,45 @@
+import magnifierIcon from "../../assets/images/magnifierIcon.png";
+import { useMoviesContext } from "../../stateManagement/store";
+import Logo from "../common/Logo/Logo";
+
+import styles from "./movieDetails.module.css";
+
+function MovieDetails() {
```

Comment:
![image](https://user-images.githubusercontent.com/106384184/208586900-fd4ae58d-50d2-40ff-8ecb-3330acbabcc3.png)

You have a layout in Figma, there is no need to make it pixel perfect, but the yours layout must match the Figmas layout

Need a font family, as well